tests/conftest.py

```diff
@@ -45,17 +45,6 @@ def IsNow(*args: Any, **kwargs: Any):
         return _IsNow(*args, **kwargs)
 
 
-try:
-    from logfire.testing import CaptureLogfire
-except ImportError:
-    pass
-else:
-
-    @pytest.fixture(autouse=True)
-    def logfire_disable(capfire: CaptureLogfire):
-        pass
```

Comment:
Unrelated to the rest, @dmontagu pointed out that this slows down tests greatly, and we have no reason to keep this.